### PR TITLE
GH-90690: Mention removal of ``PRECALL`` in What's New

### PR DESCRIPTION
**Pull request by tonybaloney**
The removal of `PRECALL` is missing from the release notes. 

https://github.com/python/cpython/pull/92925

* Issue: gh-90690

### CODE REVIEW

**Review by AA-Turner — APPROVED** [review]

File: Doc/whatsnew/3.12.rst [hunk 1]
```diff
@@ -611,6 +611,8 @@ CPython bytecode changes
 * Removed the :opcode:`!JUMP_IF_FALSE_OR_POP` and :opcode:`!JUMP_IF_TRUE_OR_POP`
   instructions. (Contributed by Irit Katriel in :gh:`102859`.)
 
+* Removed the :opcode:`PRECALL` instruction. (Contributed by Mark Shannon in
+  :gh:`92925`)
```
Comment (AA-Turner) [inline Doc/whatsnew/3.12.rst]: Minor, for consistency with the two above.
```suggestion
  :gh:`92925`.)
```

File: Doc/whatsnew/3.12.rst [hunk 2]
```diff
@@ -611,6 +611,8 @@ CPython bytecode changes
 * Removed the :opcode:`!JUMP_IF_FALSE_OR_POP` and :opcode:`!JUMP_IF_TRUE_OR_POP`
   instructions. (Contributed by Irit Katriel in :gh:`102859`.)
 
+* Removed the :opcode:`PRECALL` instruction. (Contributed by Mark Shannon in
```
Comment (AA-Turner) [inline Doc/whatsnew/3.12.rst]: I wondered why the docs build was failing... can't cross-reference a removed opcode!

```suggestion
* Removed the :opcode:`!PRECALL` instruction. (Contributed by Mark Shannon in
```

**Review by AA-Turner — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
There are also numerous new opcodes that are not mentioned here. Perhaps we should make a table similar to the one in https://docs.python.org/3.12/whatsnew/3.11.html#cpython-bytecode-changes.

**Review by furkanonder — APPROVED** [review]

**Review by hugovk — APPROVED** [review]
(Conflict resolved.)

**Comment by hugovk** [comment]
Some opcode additions have since been documented. Let's merge this, and further improvements welcome.

**Pull request merged**